.kokoro/build.sh

```diff
@@ -69,9 +69,16 @@ integration)
     RETURN_CODE=$?
```

Comment:
Should this file be part of this PR?

Reply:
No, it's part of the generated changes by running the synth tool. It should be removed before we merge it.